alembic/versions/20240425_1833_add_drs_yeild_table.py

```diff
@@ -0,0 +1,43 @@
+"""add drs yeild table
+
+Revision ID: 11252b877df4
+Revises: d1d18f24e794
+Create Date: 2024-04-25 18:33:22.716935+00:00
+
+"""
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "11252b877df4"
+down_revision = "d1d18f24e794"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "drought_resistant_seed_yield",
+        sa.Column("id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column("replicate", sa.Integer(), nullable=True),
+        sa.Column("line", sa.String(length=200), nullable=False),
+        sa.Column("planting_date", sa.Date(), nullable=False),
+        sa.Column("harvest_date", sa.Date(), nullable=False),
+        sa.Column("yield_amount", sa.Float(), nullable=False),
```

Comment:
Do we just want to have this be yield?

Reply:
yeah that sounds cool to me

Reply:
ahh, yield is not allowed as it is a keyword in python :P

Reply:
do you still want me to change it?

Reply:
Maybe crop_yield?

Reply:
okay

Reply:
Done...